services/player-service/application/src/main/java/com/kjeldsen/player/application/usecases/GetPlayerPositionTendencyUseCase.java

```diff
@@ -0,0 +1,18 @@
+package com.kjeldsen.player.application.usecases;
+
+import com.kjeldsen.player.application.repositories.PlayerPositionTendencyReadRepository;
+import com.kjeldsen.player.domain.PlayerPosition;
+import com.kjeldsen.player.domain.PlayerPositionTendency;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@AllArgsConstructor
+public class GetPlayerPositionTendencyUseCase {
```

Comment:
I think basic use cases like find, save, etc could be skipped and just access the repository itself. I do not see the benefit

Reply:
I removed all find / get use cases.